op-chain-ops/genesis/testdata/l1-deployments.json

```diff
@@ -18,5 +18,7 @@
   "SystemConfig": "0x36bAcDD96F28e1ac0780bB9CbE6e20780840730F",
   "SystemConfigProxy": "0x14065A373936533A0c88b7986CADabDD62d471e6",
   "ProtocolVersions": "0x883C06a27D76B1CEbdf7EB376f5556c355afC8e5",
-  "ProtocolVersionsProxy": "0x3732a4D4Ab006cA4825822baEA1569A107683fa1"
+  "ProtocolVersionsProxy": "0x3732a4D4Ab006cA4825822baEA1569A107683fa1",
+  "SuperchainConfig": "0x883C06a27D76B1CEbdf7EB376f5556c355afC8e5",
+  "SuperchainConfigProxy": "0x3732a4D4Ab006cA4825822baEA1569A107683fa1"
```

Comment:
These are not the correct values. I just put them in to get the correct address from the failing test.

Reply:
Yes. All it does is check that the addresses are non-zero. https://github.com/ethereum-optimism/optimism/blob/develop/op-chain-ops/genesis/testdata/l1-deployments.json#L20-L21